js/mexc3.js

```diff
@@ -0,0 +1,3659 @@
+                            'market/api_default_symbols': 2,
+                        },
+                    },
+                    'private': {
+                        'get': {
+                            'account/info': 1,
+                            'order/open_orders': 1,
+                            'order/list': 1,
+                            'order/query': 1,
+                            'order/deals': 1,
+                            'order/deal_detail': 1,
+                            'asset/deposit/address/list': 2,
+                            'asset/deposit/list': 2,
+                            'asset/address/list': 2,
+                            'asset/withdraw/list': 2,
+                            'asset/internal/transfer/record': 10,
+                            'account/balance': 10,
+                            'asset/internal/transfer/info': 10,
+                            'market/api_symbols': 2,
+                        },
+                        'post': {
+                            'order/place': 1,
+                            'order/place_batch': 1,
+                            'asset/withdraw': 2,
+                            'asset/internal/transfer': 10,
+                        },
+                        'delete': {
+                            'order/cancel': 1,
+                            'order/cancel_by_symbol': 1,
+                            'asset/withdraw': 2,
+                        },
+                    },
+                },
+            },
+            'timeframes': {
+                '1m': '1m', // spot, swap
+                '3m': '3m', // spot
+                '5m': '5m', // spot, swap
+                '15m': '15m', // spot, swap
+                '30m': '30m', // spot, swap
+                '1h': '1h', // spot, swap
+                '2h': '2h', // spot
+                '4h': '4h', // spot, swap
+                '6h': '6h', // spot
+                '8h': '8h', // spot, swap
+                '12h': '12h', // spot
+                '1d': '1d', // spot, swap
+                '3d': '3d', // spot
+                '1w': '1w', // spot, swap
+                '1M': '1M', // spot, swap
+            },
+            'fees': {
+                'trading': {
+                    'tierBased': false,
+                    'percentage': true,
+                    'maker': 0.2 / 100, // maker / taker
+                    'taker': 0.2 / 100,
+                },
+            },
+            'options': {
+                'fetchMarkets': {
+                    'types': {
+                        'spot': true,
+                        'future': {
+                            'linear': false,
+                            'inverse': false,
+                        },
+                        'swap': {
+                            'linear': true,
+                            'inverse': false,
+                        },
+                    },
+                },
+                'timeframes': {
```

Comment:
Question, why is this inside options?

Reply:
I'll mark this as resolved, as discussed above.